lib/github_changelog_generator.rb

```diff
@@ -139,7 +139,7 @@ def fetch_merged_at_pull_requests
       }
 
       if @options[:verbose]
-        puts 'Fetching merged dates... Done!'
+        puts "Fetching merged dates: Done!"
```

Comment:
I would've gone with http://viget.com/extend/just-use-double-quoted-ruby-strings and always use double quotes. But if you prefer mixed style, I can revert this part :)